Formula/reorder-python-imports.rb

```diff
@@ -0,0 +1,28 @@
+class ReorderPythonImports < Formula
+  include Language::Python::Virtualenv
+
+  desc "Rewrites source to reorder python imports"
+  homepage "https://github.com/asottile/reorder_python_imports"
+  url "https://github.com/asottile/reorder_python_imports/archive/v2.3.0.tar.gz"
+  sha256 "c544b19a2900220e6a8d1a070dd651ce0c6885ca592f4e4af99f7551b6d9807b"
+
+  depends_on "python"
+
+  resource "aspy.refactor-imports" do
+    url "https://files.pythonhosted.org/packages/34/6e/37cbfba703b06fca29c38079bef76cc01e8496197701fff8f0dded3b5b38/aspy.refactor_imports-2.1.1.tar.gz"
+    sha256 "eec8d1a73bedf64ffb8b589ad919a030c1fb14acf7d1ce0ab192f6eedae895c5"
+  end
+
+  resource "cached-property" do
+    url "https://files.pythonhosted.org/packages/57/8e/0698e10350a57d46b3bcfe8eff1d4181642fd1724073336079cb13c5cf7f/cached-property-1.5.1.tar.gz"
+    sha256 "9217a59f14a5682da7c4b8829deadbfc194ac22e9908ccf7c8820234e80a1504"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/reorder-python-imports"
```

Comment:
I see that according to this https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula I need to write a better test for this. 
I am working on that test